crates/solver/src/settlement_submission/submitter/custom_nodes_api.rs

```diff
@@ -41,14 +41,14 @@ impl TransactionSubmitting for CustomNodesApi {
         &self,
```

Comment:
Maybe also rename this file? (sorry for super nitpicking)

Reply:
I renamed the `CustomNodesApi` struct as well as the file and updated the 2 remaining log messages.